src/main.js

```diff
@@ -25,18 +25,44 @@ define((require, exports, module) => {
   // var NativeApp = brackets.getModule("utils/NativeApp");
   // var Commands = brackets.getModule("command/Commands");
 
+  const menu = Menus.getMenu(Menus.AppMenuBar.VIEW_MENU);
+
+
+  const WtaName = 'Web Tools'; // WTA = WebToolsActivation
+  const WtaID = 'bob.Web_Tools';
+  const WtaState = true;
 
-  const wtaId = 'bob.wta'; // WTA = WebToolsActivation
-  const wtaName = 'Web Tools';
   function wta() {
     // AirBnB standard does will throw errors when you leave console.log
     // console.log('Executing Command WTA');
+	if (!WtaState) {
+	  Menus.getContextMenu(Menus.ContextMenuIds.EDITOR_MENU).addSubMenu('NADCHIF_ECDC_MENU'); // Example with enc/dec 118
+      WtaState = true;
+	} else {
+      Menus.getContextMenu(Menus.ContextMenuIds.EDITOR_MENU).removeSubMenu('NADCHIF_ECDC_MENU'); // Example with enc/dec 118
+      WtaState = false;
```

Comment:
Help me understand this. Tell me if I am wrong.
If the 'Web tools' menu is clicked then a new submenu would be displayed? Is that it?